src/rust/ide/lib/view/src/graph_editor/component/breadcrumbs/project_name.rs

```diff
@@ -2,11 +2,20 @@
 
 use crate::prelude::*;
 
+use crate::graph_editor::component::breadcrumbs::TEXT_SIZE;
+use crate::graph_editor::component::breadcrumbs::GLYPH_WIDTH;
+use crate::graph_editor::component::breadcrumbs::VERTICAL_MARGIN;
+use crate::graph_editor::component::breadcrumbs::breadcrumb::HOVER_COLOR;
+use crate::graph_editor::component::breadcrumbs::breadcrumb::LEFT_MARGIN;
+use crate::graph_editor::component::breadcrumbs::breadcrumb::LEFT_DESELECTED_COLOR;
+use crate::graph_editor::component::breadcrumbs::breadcrumb::SELECTED_COLOR;
+use crate::graph_editor::component::breadcrumbs::breadcrumb::RIGHT_MARGIN;
+use crate::graph_editor::component::breadcrumbs::breadcrumb::PADDING;
+use crate::graph_editor::component::breadcrumbs::breadcrumb::TOP_MARGIN;
```

Comment:
just a small request - could you use these in qualified form instead? Just use 2 imports and access constants in qualified manner please.